src/redux/local-storage.js

```diff
@@ -24,6 +24,11 @@ export const getLocalStorageBoolean = (key, defValue) => {
     return value === 'true';
 };
 
+export const getLocalStorageInteger = (key, defValue) => {
```

Comment:
remove this function ?
- It's called getInteger but doesn't do any integer handling
- no need to define a function doing just a || b

Reply:
Done